### PR TITLE
Hid the archived tag from students

### PR DESCRIPTION
**Pull request by MitchelPaulin**
Only shows the archived tag for anyone who has permissions to either assess or edit the current rubric, fix for both the compact and full screen view.

In a previous PR to hm-api-tests I added back the status types to the titles of the rubrics, the vdiff changes are mostly adding the title back. Another change is there is no longer a `not graded yet` on some of the rubrics. I noticed the students would see this `not graded yet` on a rubric preview. Hid the graded info if no assessment href was present. 

- back end change https://github.com/Brightspace/lms/pull/22681
- hm-api-test change https://github.com/Brightspace/hm-api-tests/pull/302

https://rally1.rallydev.com/#/57732444928ud/custom/373260458992?detail=%2Fuserstory%2F633538539905

### CODE REVIEW

File: src/components/rubric.js [hunk 1]
```diff
@@ -691,21 +693,21 @@ export class Rubric extends composeMixins(
...
+		let assessorText = null;
 		const assessorName = this._assessorUserEntity?.userName;
 		if (assessorName) {
 			if (assessorName === 'unknown') {
 				assessorText = this.localize('rubricEvaluation:gradedByUnknown');
 			} else {
 				assessorText = this.localize('rubricEvaluation:gradedBy', 'name', assessorName);
 			}
-		} else {
+		} else if (this.assessmentHref) {
```
Comment (jesseag) [inline src/components/rubric.js:710]: When the component is initialized, I assume `this.assessmentHref = ''`

I would think we should display the text "not Graded yet" in that instance. In this instance, we won't display the text if the string is blank.

Comment (jesseag) [inline src/components/rubric.js:710]: When the component is initialized, I assume `this.assessmentHref = ''`

I would think we should display the text "not Graded yet" in that instance. In this instance, we won't display the text if the string is blank.

Comment (MitchelPaulin) [reply src/components/rubric.js:710]: I think even if the rubric is not graded yet, an assessment href is still present, a string may be empty on first render but its listed as a property so a render update would happen immediately after

Comment (jesseag) [reply src/components/rubric.js:710]: So the render update would guarantee the string wouldn't be empty even if no assessment has occurred?

**Review by jesseag — APPROVED** [review]
Looks good to me.

**Review by imcdonald-d2l — APPROVED** [review]

**Review by devpow112 — APPROVED** [review]

**Pull request merged**